app/versioning.py

```diff
@@ -0,0 +1,62 @@
+from typing import Callable, List
+
+import flask
+from werkzeug.exceptions import BadRequest
+
+VALID_API_VERSIONS = ['1.0']
```

Comment:
I know we talked about a list before, but thinking more about it, do you think this should be a tuple instead so it isn't mutable?

Reply:
That makes complete sense.

Reply:
Ok, once you make this change I can do another round of testing and see if I find any problems with it. Otherwise, looking good

Reply:
- [ ] change list of API versions to a tuple of API versions